.github/workflows/ci-test.yaml

```diff
@@ -17,46 +17,29 @@ jobs:
         ports:
           - 8545:8545
           - 8546:8546
-        env:
-          OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
-        options: >-
-          --rm
-          --health-cmd="test -f /CONTAINER_READY"
-          --health-start-period=90s
+        # env:
+        #   OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
+        # options: >-
+        #   --rm
+        #   --health-cmd="test -f /CONTAINER_READY"
+        #   --health-start-period=90s
     steps:
     - name: Check out repository
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
 
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: '3.12'
 
-    - name: Checkout tools repo
-      uses: actions/checkout@v4
-      with:
-        repository: https://github.com/oasisprotocol/sapphire-paratime.git
-        path: /wrapper/
-
-    - name: Build sapphirepy .whl file
-      working-directory: /wrapper/sapphire-paratime/clients/py
+    - name: Install dependencies
       run: |
-        pip3 install --user -r requirements.txt
-        pip3 install --user -r requirements.dev.txt
         make

```

Comment:
Installing dependencies with `make` seems wrong?

Reply:
Also, I suggest we pin `oasis-sapphire-py=0.4.0` in `requirements.txt`.

Reply:
> Also, I suggest we pin `oasis-sapphire-py=0.4.0` in `requirements.txt`.

Even better, this should be part of #6.

Reply:
Removed make, added pip3 command. 
Will update the version in #6